PyAlbany/src/Albany_Interface.cpp

```diff
@@ -434,6 +434,8 @@ Teuchos::RCP<PyTrilinosMultiVector> PyProblem::getReducedHessian(const int g_ind
 
 void PyProblem::performSolve()
 {
+    if (Teuchos::TimeMonitor::getStackedTimer() != stackedTimer)
+        Teuchos::TimeMonitor::setStackedTimer(stackedTimer);
```

Comment:
Naive question, why do you need to check whether the stacked timers are the same, why don't you simply set it?

Reply:
I thought that it would be best not to set the stackedTimer if it was already set correctly. But I have no strong feelings about that to be honest.